bin/getCryptoData.js

```diff
@@ -0,0 +1,157 @@
+import path from 'path';
```

Comment:
Hmm... why is this file showing up? This file was merged into master a couple week ago.

Reply:
Crud. Updating from master must have screwed something up.